### ISSUE TITLE
Unable to generate output other than html via template parameter with extension other than html

### ISSUE THREAD

**Issue by tombreit** [thanethomson/statik#47]
The following view/template combo does not produce a the desired json file, instead yields a 

`MissingTemplateError('Cannot find template by name: search_index.json',)`:

```
# views/search_index.yml
path: /search_index.json
template: search_index.json
context:
  static:
    page-title: Search Index
  dynamic:
    photos: session.query(Photo).all()
    albums: session.query(Album).all()
```

----

```
# templates/search_index.json
does-not-matter
```

What am I doing wrong? I'd like to generate a json file and name the template file accordingly. When switching the file extension to `.html` everything works as expected.

**Comment by thanethomson** [comment]
Yup, this is definitely a bug. Looking into it now.

**Comment by thanethomson** [comment]
Now fixed in v0.18.1 😁
